readme.md

```diff
@@ -0,0 +1,101 @@
+# Fashgram Store - where creative meets Fashion
+
```

Comment:
Readme looks well documented, Suggestion, you could add a video maybe showing your quiz app.